shared/folders/files/render.native.js

```diff
@@ -25,37 +25,46 @@ class FilesRender extends Component<void, Props, void> {
 
   // TODO render checkerboard pattern for private mode
   _renderHeader () {
-    const menuColor = styleMenuColorThemed(this.props.theme, this.props.visiblePopupMenu)
     const backButtonColor = backButtonColorThemed[this.props.theme]
 
     const contents = (
       <Box style={{...globalStyles.flexBoxRow, justifyContent: 'space-between', ...styleHeaderThemed[this.props.theme], height: 48}}>
         <BackButton title={null} onClick={this.props.onBack} style={{marginLeft: globalMargins.small}} iconStyle={{color: backButtonColor}} textStyle={{color: backButtonColor}} />
-        <Icon
-          underlayColor={'transparent'}
-          style={{...styleMenu, color: menuColor, marginRight: globalMargins.small}}
-          type='iconfont-hamburger'
-          onClick={this.props.onTogglePopupMenu} />
       </Box>
     )
 
     return contents
   }
 
+  _renderIgnoreButton (isPrivate: boolean) {
+    return this.props.ignored
+      ? <Button type='Secondary' onClick={this.props.unIgnoreCurrentFolder} label='Unignore folder' />
+      : <Button backgroundMode={isPrivate ? 'Terminal' : 'Normal'} type='Secondary' onClick={this.props.ignoreCurrentFolder} label='Ignore folder' />
+  }
+
   _renderContents (isPrivate: boolean, ignored: boolean, allowIgnore: boolean) {
     if (!this.props.recentFilesEnabled) {
       return (
         <Box style={{...globalStyles.flexBoxColumn, flex: 1, justifyContent: 'center', alignItems: 'center'}}>
-          {ignored
-          ? allowIgnore && <Button type='Secondary' onClick={this.props.unIgnoreCurrentFolder} label='Unignore folder' />
-          : allowIgnore && <Button backgroundMode={isPrivate ? 'Terminal' : 'Normal'} type='Secondary' onClick={this.props.ignoreCurrentFolder} label='Ignore folder' />}
+          {allowIgnore && this._renderIgnoreButton(ignored)}
         </Box>
       )
     } else {
       return <NativeScrollView>{this.props.recentFilesSection.map(s => this._renderSection(s))}</NativeScrollView>
     }
   }
 
+  _renderNotImplemented (isPrivate: boolean) {
```

Comment:
i know this file is very old but we should lean towards using a stateful component and not deferring to these helper render functions
```<NotImplemented isPrivate={true}/>```